modules/olm-installing-po-during.adoc

```diff
@@ -0,0 +1,112 @@
+// Module included in the following assemblies:
+//
+// * operators/admin/olm-managing-po.adoc
+
+:_content-type: PROCEDURE
+[id="olm-installing-po-during_{context}"]
+= Installing platform Operators during cluster creation
+
+As a cluster administrator, you can install platform Operators by providing `FeatureGate` and `PlatformOperator` manifests during cluster creation.
+
+.Procedure
+
+. Choose a platform Operator from the supported set of OLM-based Operators. For the list of this set and details on current limitations, see "Technology Preview restrictions for platform Operators".
+
+. Select a cluster installation method and follow the instructions through creating an `install-config.yaml` file. For more details on preparing for a cluster installation, see "Selecting a cluster installation method and preparing it for users".
+
+. After you have created the `install-config.yaml` file and completed any modifications to it, change to the directory that contains the installation program and create the manifests:
++
+[source,terminal]
+----
+$ ./openshift-install create manifests --dir <installation_directory> <1>
+----
+<1> For `<installation_directory>`, specify the name of the directory that contains the `install-config.yaml` file for your cluster.
+
+. Create a `FeatureGate` object YAML file in the `<installation_directory>/manifests/` directory that enables the `TechPreviewNoUpgrade` feature set, for example a `feature-gate.yaml` file:
++
+.Example `feature-gate.yaml` file
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: FeatureGate
+metadata:
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+    release.openshift.io/create-only: "true"
+  name: cluster
+spec:
+  featureSet: TechPreviewNoUpgrade <1>
+----
+<1> Enable the `TechPreviewNoUpgrade` feature set.
+
+. Create a `PlatformOperator` object YAML file for your chosen platform Operator in the `<installation_directory>/manifests/` directory, for example a `cert-manager.yaml` file for the {cert-manager-operator}:
++
+.Example `cert-manager.yaml` file
+[source,yaml]
+----
+apiVersion: platform.openshift.io/v1alpha1
+kind: PlatformOperator
+metadata:
+  name: cert-manager
+spec:
+  package:
+    name: openshift-cert-manager-operator
+----
+
+. When you are ready to complete the cluster install, refer to your chosen installation method and continue through running the `openshift-install create cluster` command.
```

Comment:
Maybe we should add a note to remind the user that the failure of the platform operator creation will block the cluster's installation.